Components/PackList.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { View, FlatList, StyleSheet, Text } from 'react-native';
+import packlist from '../.expo-shared/packlist.json';
+import Unorderedlist from 'react-native-unordered-list';
+import styled from 'styled-components/native';
+
+export const PackList = () => {
+  return (
+    <Container>
+      <Emojis> 🌲 🌲 🌲 </Emojis>
+      <InnerContainer>
+        <Text>
+          <Title>Packlist</Title>
+          {'\n'}
```

Comment:
Is this a way to make a line break? Haven't seen this before.

Reply:
Yes that's right, it's to make a break. I had to google it up, because the </br> don't work in react native.